packages/desktop-client/config-overrides.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
         generateStatsFile: true,
       }),
     ),
+    // Pikaday throws a warning if Moment.js is not installed however it doesn't
+    // actually require it to be installed. As we don't use Moment.js ourselves
+    // then we can just silence this warning.
+    addWebpackPlugin(
+      new IgnorePlugin({
+        contextRegExp: /moment$/,
+        resourceRegExp: /pikaday$/,
+      }),
+    ),
```

Comment:
Same patch as here: https://github.com/actualbudget/actual/pull/131/files

Not great.. but slightly better than monkeypatch.